wandb/sdk/artifacts/artifact.py

```diff
@@ -2098,13 +1946,14 @@ def link(self, target_path: str, aliases: Optional[List[str]] = None) -> None:
         """Link this artifact to a portfolio (a promoted collection of artifacts).
 
         Arguments:
-            target_path: The path to the portfolio. It must take the form {portfolio},
-                {project}/{portfolio} or {entity}/{project}/{portfolio}.
-            aliases: A list of strings which uniquely identifies the artifact inside the
+            target_path: The path to the portfolio. The target path must
+            adhere to one of the following schemas `{portfolio}`,
+            `{project}/{portfolio}` or `{entity}/{project}/{portfolio}`.
```

Comment:
```suggestion
            `{project}/{portfolio}` or `{entity}/{project}/{portfolio}`. To link it the Model Registry, rather than to a generic portfolio inside a project, the `target_path` should adhere to the following schema: `{"model-registry"}/{Registered Model Name}` or `{entity}/{"model-registry"}/{Registered Model Name}`. 
```

Reply:
@ibindlish is the target path for liking to MR correct here?

Reply:
Yes